src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2454,6 +2467,10 @@ export class ResultsViewPageStore {
         return new GeneCache();
     }
 
+    @cached get genesetCache() {
```

Comment:
where is this being used (the cache)?  We are actually moving away from using caches like this so I wouldn't bother with this unless there's something i'm misunderstanding.    We have implemented a client level cache (not merged yet) that will take care of this in a more abstract way.

Reply:
It is used in line 1686 of this same file to retrieve the gene sets, similarly as done with genes.

Reply:
but where is it actually consumed by view layer?  like where is cache data actually accessed?